content/docs/deployment.md

```diff
@@ -212,14 +225,21 @@ An example configuration file:
         }
       ]
     }
-  ],
-  "dependenciesMenuEnabled": false
+  ]
 }
 ```
 
-In the above example, `gaTrackingID` will be used as the Google Analytics tracking ID and `menu` configures the menu in the top right of the UI.
+`dependencies.dagMaxNumServices` defines the maximum number of services allowed before the DAG dependency view is disabled. Default: `200`.
+
+`dependencies.menuEnabled` enables (`true`) or disables (`false`) the dependencies menu button. Default: `true`.
+
+`tracking.gaID` defines the Google Analytics tracking ID. This is required for Google Analytics tracking, and setting it to a non-`null` value enables Google Analytics tracking. Default: `null`.
+
+`tracking.trackErrors` enables (`true`) or disables (`false`) error tracking via Google Analytics. Errors can only be tracked if a valid Google Analytics ID is provided. For additional details on error tracking via Google Analytics see the [tracking README](https://github.com/jaegertracing/jaeger-ui/blob/c622330546afc1be59a42f874bcc1c2fadf7e69a/src/utils/tracking/README.md) in the UI repo. Default: `true`.
```

Comment:
should the link to the README be via concrete hash or just via master branch?

Reply:
I hate hitting dead-links and this is a great way to avoid it. I'm fine either way.